CorsixTH/Lua/entities/humanoid.lua

```diff
@@ -642,6 +592,65 @@ function Humanoid:hasLeavingAction()
   return false
 end
 
+--! Handle an empty action queue in some way instead of crashing.
+function Humanoid:handleEmptyActionQueue()
+  -- if this is a patient that is going home, an empty
+  -- action queue is not a problem
+  if class.is(self, Patient) and self.going_home then
+    return
+  end
+
+  -- First find out if this humanoid is in a room.
+  local room = self:getRoom()
+  if room then
+    room:makeHumanoidLeave(self)
+  end
+
+  -- Give the humanoid an action to avoid crashing.
+  if class.is(self, Staff) then
+    self:queueAction(MeanderAction())
+  elseif class.is(self,GrimReaper) then
```

Comment:
```suggestion
  elseif class.is(self, GrimReaper) then
```
(sorry, was bothering me)

Reply:
done